TourMate/TourMate/Frontend/Plan/Views/PlanUpvoteView/PlanUpvoteView.swift

```diff
@@ -9,11 +9,11 @@ import SwiftUI
 
 struct PlanUpvoteView: View {
 
-    @StateObject var viewModel: PlanUpvoteViewModel
+    @ObservedObject var viewModel: PlanUpvoteViewModel
```

Comment:
i think all views using this would have to change to stateobject before passing creating this

Reply:
Ok I updated it